### PR TITLE
optimize match node label

### PR DESCRIPTION
**Pull request by nevermore3**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: src/graph/executor/query/TraverseExecutor.cpp [hunk 1]
```diff
@@ -164,27 +160,18 @@ folly::Future<Status> TraverseExecutor::handleResponse(RpcResponse&& resps) {
...
-    existVids.reserve(iter->numRows());
+    initVertices_.reserve(iter->numRows());
     auto vertices = iter->getVertices();
+    // match (v)-[e:Rel]-(v1:Label1)-[e1*2]->() where id(v0) in [6, 23] return v1
+    // save the vertex that meets the filter conditions as the starting vertex of the current
+    // traverse
     for (auto& vertex : vertices.values) {
       if (vertex.isVertex()) {
-        existVids.emplace(vertex);
+        initVertices_.emplace_back(vertex);
```
Comment (yixinglu) [inline src/graph/executor/query/TraverseExecutor.cpp:170]: Isn't it necessary to deduplicate the vids for next traverse ?

Comment (nevermore3) [reply src/graph/executor/query/TraverseExecutor.cpp:170]: the first column vid in the result returned by getNeighbor is not repeated

Comment (yixinglu) [reply src/graph/executor/query/TraverseExecutor.cpp:170]: make sense.

**Review by yixinglu — APPROVED** [review]
LGTM

**Review by codesigner — APPROVED** [review]

**Pull request merged**